libmscore/mscore.cpp

```diff
@@ -375,6 +375,17 @@ void MScore::init()
                         exit(-1);
                   }
             }
+#endif
+// Workaround for QTBUG-73241 (solved in Qt 5.12.2, see https://musescore.org/en/node/280244
+#if defined(Q_OS_WIN) && (QT_VERSION < QT_VERSION_CHECK(5, 12, 2))
```

Comment:
We could even check for Windows 10 here (or  `if (QSysInfo::WindowsVersion <= QSysInfo::WV_WINDOWS7)`)?
Or doesn't this have any bad effect on and Windows prior to Windows 10 1809?

Reply:
It seems that QSysInfo::WindowsVersion is an obsolete member of Qt. I don't know how it will work when newer versions of Windows will appear.
At the moment, for  Windows versions these lines are adding to MuseScore all fonts possibly present in C:\Users\UserName\AppData\Local\Microsoft\Windows\Fonts.

Reply:
Which might add fonts that MuseScore sees without that workaround already prior to Windows 10 1809? Would that create duplicates, or just waist few CPU cycles?

Reply:
The newer version is `if (QOperatingSystemVersion::current() <= QOperatingSystemVersion(QOperatingSystemVersion::Windows, 7))`

Reply:
For what concerns duplicates, it depends on how Qt handles them.
Prior to Windows 10 1809 fonts could be installed only in system location C:\Windows\Fonts.
This workaround adds also all the fonts present in C:\Users\UserName\AppData\Local\Microsoft\Windows\Fonts. For duplicates *maybe* the behavior could be similar to what we see when users install MuseScore internal fonts at system level (i.e. the font chosen by the Application could be one or the other); however, in case of duplicates, they have deliberately chosen to install those duplicates in the user font folder (and they can freely uninstall them).

By the way, I think I found a way to check Windows version with non-deprecated functions.